src/library/scala/io/Source.scala

```diff
@@ -167,6 +167,10 @@ object Source {
 
   def fromInputStream(is: InputStream)(implicit codec: Codec): BufferedSource =
     createBufferedSource(is, reset = () => fromInputStream(is)(codec), close = () => is.close())(codec)
+
+  def fromClassPath(resource: String, classLdr: ClassLoader = ClassLoader.getSystemClassLoader)(implicit codec: Codec): BufferedSource =
```

Comment:
I was kind of kidding about the braces, but that is good spotting. Now I know why they call him Hawkeye; or will call him that in times to come. Unfortunately, this will deprive soc of a deprecatedName fix with removal in 2.13.